### ISSUE TITLE
Crawler gets stuck on PDP (I think)

### ISSUE THREAD

**Issue by miskoll** [SirSharpest/blackboard_crawler#8]
Hi Nathan, 

I'm not sure if this is just a problem on my end but I've tried running the crawler on different OSes and I get the same problem.  From what I can tell, the crawler goes into the DEPT-N-UG-COMPUTER-SCIENCE module page and gets stuck on the PDP section.  It lists the modules and then outputs the following:

```
looking at: Content
looking at: Link to PDP on Dept-N-UG module
Traceback (most recent call last):
  File "urllib/request.py", line 1243, in do_open
  File "http/client.py", line 1106, in request
  File "http/client.py", line 1151, in _send_request
  File "http/client.py", line 1102, in endheaders
  File "http/client.py", line 934, in _send_output
  File "http/client.py", line 877, in send
  File "http/client.py", line 1252, in connect
  File "http/client.py", line 849, in connect
  File "socket.py", line 693, in create_connection
  File "socket.py", line 732, in getaddrinfo
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 260, in <module>
  File "main.py", line 202, in populate_modules
  File "main.py", line 193, in get_all_folders
  File "main.py", line 182, in get_all_folders
  File "main.py", line 98, in get_folder_links
  File "urllib/request.py", line 162, in urlopen
  File "urllib/request.py", line 465, in open
  File "urllib/request.py", line 483, in _open
  File "urllib/request.py", line 443, in _call_chain
  File "urllib/request.py", line 1286, in https_open
  File "urllib/request.py", line 1245, in do_open
urllib.error.URLError: <urlopen error [Errno -2] Name or service not known>
Failed to execute script main
```
This was run on Ubuntu 14.04 using both the executable in `dist/main` and the `main.py` script itself, but it still throws the same error.

I'd have a go at fixing it myself but Python's not my forte, I'm afraid!

**Comment by SirSharpest** [comment]
I'm quite busy with work this week but I'll try and have a look at it by this weekend and get back to you. It could be a possible issue with blackboard changing link names, or more likely that folder has some icky styled links! 

If you're okay with python and wanna have a look I'd suggest making an edit something along the lines of "for link in links: if "DEPT-N-UG" in link continue" or something like that to test if the rest of the program still works.

**Comment by miskoll** [comment]
This issue is fixed as of commit afdf464.  Thanks!

/cc @cgddrd
